mozdownload/cli.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from optparse import OptionParser, OptionGroup
+import os
+import pkg_resources
+import sys
+
+from . import errors
+from . import scraper
+
+
+version = pkg_resources.require("mozdownload")[0].version
+
+__doc__ = """
+Module to handle downloads for different types of archive.mozilla.org hosted \
```

Comment:
Just to add... reason why those instances are present is that I only copy and pasted the code from one file to the other to minimize code changes for the reviewer. Sorry, that I didn't explicitly mention that.